### PR TITLE
chore: create Prometheus quickstart description

### PR DESCRIPTION
**Pull request by aliandersonsn**
Search Nurture recommendations for Prometheus description and summary

## Summary

A concise description of the changes being introduced.
If this is a new Observability Pack, please provide a description of its use cases and a list of the Pack's components.

### Screenshots

Images of any visual changes, such as a dashboard.

### CODE REVIEW

File: packs/prometheus/config.yml [hunk 1]
```diff
@@ -1,19 +1,42 @@
...
+
+  New Relic's Prometheus Integration stores all kinds of telemetry data – whether open-source, vendor-specific, or vendor-agnostic. 
+
+  ### Value of the Prometheus Pack 
+
+  New Relic’s quickstart features a myriad of options to make DevOps easier. Although there are many ways to use Prometheus data in New Relic, we’ll break these down into OpenMetrics and Remote Write to help you decide on the best option for you:
+
+  #### OpenMetrics:
+  
+  - Build customized dashboards to monitor Kubernetes HPA capacity
```
Comment (polfliet) [inline packs/prometheus/config.yml:27]: Kubernetes HPA and Node readiness are good examples of OpenMetrics data.
But there is a lot more, should we add a more generic statement here that these are just examples and any type of OpenMetrics endpoint can be instrumented?

**Review by polfliet — CHANGES_REQUESTED** [review]

**Comment by polfliet** [comment]
@aliandersonsn @bradleycamacho looks like the description is exceeding max chars (2000) as well as the summary (250)

**Comment by aswanson-nr** [comment]
The repository has been renamed to `newrelic/newrelic-quickstarts`. Github will handle routing your remotes to this new url, but we recommend that you update your remote as soon as possible.
**NOTE**: As part of this change, we have moved all the quickstarts from the `/packs` directory to the `/quickstarts` directory. To bring your pull request up to date, please pull in the latest commits from the `main` branch.

Use the following command to set your new remote:
```bash
git remote set-url origin https://github.com/newrelic/newrelic-quickstarts.git
```

Or with an SSH origin
```bash
git remote set-url origin git@github.com:newrelic/newrelic-quickstarts.git
```

File: packs/prometheus/config.yml (hunk 1 above)
Comment (bradleycamacho) [reply packs/prometheus/config.yml:27]: I added a generic statement and made the examples a bit more specific to the quickstart.

**Comment by bradleycamacho** [comment]
> @aliandersonsn @bradleycamacho looks like the description is exceeding max chars (2000) as well as the summary (250)

Resolved in the most recent commit

**Review by bradleycamacho — APPROVED** [review]

**Comment by polfliet** [comment]
Closing, these changes were merged as part of https://github.com/newrelic/newrelic-quickstarts/pull/357

**Pull request closed without merging**